.github/workflows/clean-images.yaml

```diff
@@ -0,0 +1,38 @@
+name: "PR checks - Clean operator images"
+
+on:
+  pull_request_target:
+    types: [closed]
+
+jobs:
+  clean-operator-images:
+    name: "Clean operator images (PR)"
+    runs-on: ubuntu-22.04
+    strategy:
+      matrix:
+        include:
+          - image_base: host-operator
+          - image_base: host-operator-bundle
+          - image_base: host-operator-index
+          - image_base: member-operator
+          - image_base: member-operator-bundle
+          - image_base: member-operator-console-plugin
+          - image_base: member-operator-index
+          - image_base: member-operator-webhook
+          - image_base: registration-service
+          - image_base: workspace-operator
+          - image_base: workspace-server
```

Comment:
do you think we can make this dynamic with `skopeo`? like all repositories under `konflux-workspaces` org?

Reply:
`skopeo` gives us `list-tags`, but not a generic `list` command.  I'm not aware of a method for it to list all repositories under a namespaces; you might have to talk to the quay.io api to get that to work, and this just seemed easier.